css/properties/line-height-step.json

```diff
@@ -0,0 +1,54 @@
+{
+  "css": {
+    "properties": {
+      "line-height-step": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/line-height-step",
+          "support": {
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": "59"
```

Comment:
In Chrome and Opera this is still behind a runtime flag. https://www.chromestatus.com/features/5734273533345792

Reply:
Thanks, fixed in 5f968da9a25013b6b4d3cae15d090ba0f5189323.